docs/sources/http_api/serviceaccount.md

```diff
@@ -0,0 +1,341 @@
+	"page": 1,
+	"perPage": 10
+}
+```
+
+## Create service account
+
+`POST /api/serviceaccounts`
+
+#### Required permissions
+
+See note in the [introduction]({{< ref "#serviceaccount-api" >}}) for an explanation.
+
+| Action                | Scope              |
+| --------------------- | ------------------ |
+| serviceaccounts:write | serviceaccounts:\* |
+
+**Example Request**:
+
+```http
+POST /api/serviceaccounts HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Basic YWRtaW46YWRtaW4=
+
+{
+  "name": "grafana",
+  "role": "Admin",
+}
+```
+
+Requires basic authentication and that the authenticated user is a Grafana Admin.
+
+**Example Response**:
+
+```http
+HTTP/1.1 200
+Content-Type: application/json
+
+{
+	"id": 1,
+	"name": "test",
+	"login": "sa-test",
+	"orgId": 1,
+	"isDisabled": false,
+	"createdAt": "2022-03-21T14:35:33Z",
+	"updatedAt": "2022-03-21T14:35:33Z",
+	"avatarUrl": "/avatar/8ea890a677d6a223c591a1beea6ea9d2",
+	"role": "Viewer",
+	"teams": []
+}
+```
+
+## Get single serviceaccount by Id
+
+`GET /api/serviceaccounts/:id`
+
+#### Required permissions
+
+See note in the [introduction]({{< ref "#serviceaccount-api" >}}) for an explanation.
+
+| Action               | Scope              |
+| -------------------- | ------------------ |
+| serviceaccounts:read | serviceaccounts:\* |
+
+**Example Request**:
+
+```http
+GET /api/serviceaccounts/1 HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Basic YWRtaW46YWRtaW4=
+```
+
+Requires basic authentication and that the authenticated user is a Grafana Admin.
+
+**Example Response**:
+
+```http
+HTTP/1.1 200
+Content-Type: application/json
+
+{
+	"id": 1,
+	"name": "test",
+	"login": "sa-test",
+	"orgId": 1,
+	"isDisabled": false,
+	"createdAt": "2022-03-21T14:35:33Z",
+	"updatedAt": "2022-03-21T14:35:33Z",
+	"avatarUrl": "/avatar/8ea890a677d6a223c591a1beea6ea9d2",
+	"role": "Viewer",
+	"teams": []
+}
+```
+
+## Update service account
+
+`PATCH /api/serviceaccounts/:id`
+
+#### Required permissions
+
+See note in the [introduction]({{< ref "#serviceaccount-api" >}}) for an explanation.
+
+| Action                | Scope              |
+| --------------------- | ------------------ |
+| serviceaccounts:write | serviceaccounts:\* |
+
+**Example Request**:
+
+```http
+PUT /api/serviceaccounts/2 HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Basic YWRtaW46YWRtaW4=
+
+{
+  "name": "test",
+	"role": "Editor"
+}
+```
+
+Requires basic authentication and that the authenticated user is a Grafana Admin.
+
+**Example Response**:
+
+```http
+HTTP/1.1 200
+Content-Type: application/json
+
+{
+	"id": 2,
+	"name": "test",
+	"login": "sa-grafana",
+	"orgId": 1,
+	"isDisabled": false,
+	"createdAt": "2022-03-21T14:35:44Z",
+	"updatedAt": "2022-03-21T14:35:44Z",
+	"avatarUrl": "/avatar/8ea890a677d6a223c591a1beea6ea9d2",
+	"role": "Editor",
+	"teams": []
+}
+```
+
+---
+
+## Service account tokens
+
```

Comment:
This is an example of a stacked heading, which we want to avoid. There should be some sort of intro text after the H2.